File: Bridge/Resources/.generated/system/runtime/interopServices/structLayoutAttribute.js
    Bridge.define("System.Runtime.InteropServices.StructLayoutAttribute", {
        inherits: [System.Attribute],
        fields: {
            pack: 0,
            size: 0
        },
        props: {
            Value: {
                get: function () {
                    return 0;
                }
            }
        },
        ctors: {
            ctor: function (layoutKind) {
                this.$initialize();
                System.Attribute.ctor.call(this);
                return;
            },
            $ctor1: function (layoutKind) {
                this.$initialize();
                System.Attribute.ctor.call(this);
                return;
            }
        }
    });

File: Bridge/System/Runtime/InteropServices/StructLayoutAttribute.cs
using Bridge;

namespace System.Runtime.InteropServices
{
    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Struct, Inherited = false)]
    public sealed class StructLayoutAttribute : Attribute
    {
        public int Pack;
        public int Size;
        
        public StructLayoutAttribute(short layoutKind)
        {
            return;
        }

        public StructLayoutAttribute(LayoutKind layoutKind)
        {
            return;
        }

        public LayoutKind Value
        {
            get { return default(LayoutKind); }
        }

    }
}